README.md

```diff
@@ -1,2 +1,61 @@
-pip install music21
-sudo apt-get install timidity musescore
+# Music Tutorial
+music_tutorial aims to (TBC)
+
+
+## Prerequisites:
+
+### Linux:
+Installation of musescore (for scoring) and timidity (for MIDI playback).  For most distributions using apt, the following command will install these:
+
+`sudo apt-get install timidity musescore`
+
+### macOS:
+Installation of musescore (for scoring) and VLC (for MIDI playback).
+
+Musescore: [https://musescore.org/en/download]
+
+VLC: [https://www.videolan.org/vlc/]
+
+### Windows:
+Installation of musescore (for scoring) - Windows Media Player is used for MIDI playback and is installed by default on Windows.
+
+Musescore: [https://musescore.org/en/download]
+
+## Installation:
+
+Clone the repository:
+
+`git clone [https://github.com/geokala/music_tutorial.git]`
+
+Use pip to install the requirements included in the requirements file (currently music21)
+
+`pip install -r requirements.txt`
+
+## Usage:
+
+`m210run <filename> <mode>`
+
+where
+
+`filename` - the music_tutorial file to run (such as 1/scale1.py)
+
+`mode` is one of:
+
+"midi" to hear a MIDI version of the file
+"score" to see the score
+"text" will show the python objects that the file generates
```

Comment:
It would be good to put a full example after this, e.g.
m210run 1/scale1.py midi

Reply:
Definitely - I meant to put something like that in there, but forgot - I think the example you give would be ideal as it should work straight away and be a good test.